### DISCUSSION TITLE
Can't access to SWS from outside

### DISCUSSION THREAD

**Discussion by bubnenkoff** [static-web-server/static-web-server#121] [Q&A] [answered]
Hi! Back to problem that SWS works only on localhost, and I can't open serving webpage from outside.
I checked windows ports. All fine. I even killed apps that used another port and rerun sws on it with command:
`static-web-server --cors-allow-origins "*" --port 5001 --root D:/code/test`

I can open serving page only on localhost, not from outside.

Any ideas?

**Comment by joseluisq** [comment, marked as answer]
Did you take [my comment](https://github.com/joseluisq/static-web-server/issues/119#issuecomment-1170443639) into your account?

You can serve content with SWS in your Windows network.

To do it follow those steps:

- Configure an [Inbound Port Rule](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-firewall/create-an-inbound-port-rule) in your Windows firewall so clients will be able to reach the server port.
- In your SWS config file use the server IP as a host or a non-routable address like `0.0.0.0` if you prefer.
- Finally create a Windows Service via https://sws.joseluisq.net/features/windows-service/ and start it.

Yesterday I tested that and it worked without issues.

**Comment by bubnenkoff** [comment]
Oh... it seems that issue in:
```
host = "::"
```
What does it mean? How defaults works? Should I change it to `0.0.0.0`?

Yes, setting to `0.0.0.0` helped. Please add mention to docs and to example config file.

**Reply by joseluisq** [reply]
> What does it mean? How defaults works? Should I change it to 0.0.0.0?

It means an _unspecified address_ but that is just a "save" host default. However, you can **always** change it if you need to.
